<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta
      name="viewport"
      content="width=device-width, initial-scale=1, shrink-to-fit=no"
    />
    <meta name="description" content="Pyomancer by iGEM IIT Roorkee" />
    <meta name="author" content="Team iGEM IIT Roorkee" />
    <meta name="theme-color" content="#272727">
    <title>PYOMANCER</title>

    <!-- Bootstrap core CSS -->
    <link href="../css/bootstrap.min.css" rel="stylesheet" />

    <!-- Custom styles for this template -->
    <link href="../css/igem_reset.css" rel="stylesheet" />
    <link href="../css/styles.css" rel="stylesheet" />
  </head>

  <body>
    <header>
      <div class="navbar-overlay navbar-overlay-1"></div>
      <div class="navbar-overlay navbar-overlay-2"></div>
      <nav class="navbar navbar-expand-md navbar-dark fixed-top wiki-navbar">
        <a class="navbar-brand" href="../">PYOMANCER</a>
        <button
          class="navbar-toggler"
          type="button"
          data-toggle="collapse"
          data-target="#navbarCollapse"
          aria-controls="navbarCollapse"
          aria-expanded="false"
          aria-label="Toggle navigation"
        >
          <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarCollapse">
          <ul class="navbar-nav ml-auto tabs-wrapper">
            <li class="nav-item dropdown">
                <a
                  class="tab nav-link"
                  href="#"
                  data-toggle="dropdown"
                  aria-haspopup="true"
                  aria-expanded="false"
                >
                  People
                </a>
                <ul class="dropdown-menu">
                  <li>
                    <a class="dropdown-item" href="../Team/"><span>Team</span></a>
                  </li>
                  <li>
                    <a class="dropdown-item" href="../Attributions/"><span>Attributions</span></a>
                  </li>
                  <li>
                    <a class="dropdown-item" href="../Sponsors/"><span>Sponsors</span></a>
                  </li>
                </ul>
              </li>
            <li class="nav-item dropdown">
              <a
                class="tab nav-link"
                href="#"
                data-toggle="dropdown"
                aria-haspopup="true"
                aria-expanded="false"
              >
                Project
              </a>
              <ul class="dropdown-menu">
                <li><a class="dropdown-item" href="../Description/"><span>Description</span></a></li>
                <li><a class="dropdown-item" href="../Problem/"><span>Problem</span></a></li>
                <li><a class="dropdown-item" href="../Design/"><span>Design</span></a></li>
                <li><a class="dropdown-item" href="../Engineering/"><span>Engineering</span></a></li>
                <li><a class="dropdown-item" href="../Implementation/"><span>Implementation</span></a></li>
                <li><a class="dropdown-item" href="../Contribution/"><span>Contributions</span></a></li>
              </ul>
            </li>
            <li class="nav-item dropdown">
              <a
                class="tab nav-link"
                href="#"
                data-toggle="dropdown"
                aria-haspopup="true"
                aria-expanded="false"
                >Human Practices
              </a>
              <ul class="dropdown-menu">
                <li><a class="dropdown-item" href="../Human_Practices/"> <span>Overview</span></a></li>
                <li><a class="dropdown-item" href="../Education/"> <span>Science Communication & Public Engagement</span> </a></li>
                <li><a class="dropdown-item" href="../Integrated_HP/"> <span>Integrated Human Practices</span> </a></li>
                <li><a class="dropdown-item" href="../Sustainable/"> <span>Sustainable Development Goals</span> </a></li>
                <li><a class="dropdown-item" href="../Collaborations/"> <span>Collaborations</span> </a></li>
              </ul>
            </li>
            <li class="nav-item dropdown">
              <a class="tab nav-link"
              href="#"
              data-toggle="dropdown"
              aria-haspopup="true"
              aria-expanded="false"
              >Dry Lab
              </a>
              <ul class="dropdown-menu">
              <li><a class="dropdown-item" href="../Model/"> <span>Modelling</span> </a></li>
              <li><a class="dropdown-item" href="../DryLab_Results/"> <span>Results</span> </a></li>
              <li><a class="dropdown-item" href="../Software/"> <span>Software</span> </a></li>
              </ul>
           </li>
           <li class="nav-item dropdown">
            <a class="tab nav-link"
            href="#"
            data-toggle="dropdown"
            aria-haspopup="true"
            aria-expanded="false"
            >Wet Lab
            </a>
            <ul class="dropdown-menu">
            <li><a class="dropdown-item" href="../Experiments/"> <span>Overview</span></a></li>
            <li><a class="dropdown-item" href="../Results/"> <span>Results</span> </a></li>
            <li><a class="dropdown-item" href="../Protocols/"> <span>Protocols</span> </a></li>
            <li><a class="dropdown-item" href="../Parts/"> <span>Parts</span> </a></li>
            <li><a class="dropdown-item" href="../Safety/"> <span>Safety</span> </a></li>
            <li><a class="dropdown-item" href="../References/"> <span>References</span> </a></li>
            </ul>
         </li>
         <li class="nav-item dropdown">
          <a class="tab nav-link nav-link-active"
          href="#"
          data-toggle="dropdown"
          aria-haspopup="true"
          aria-expanded="false"
          >Machine Learning
          </a>
          <ul class="dropdown-menu">
          <li><a class="dropdown-item" href="../ML_Overview/"> <span>Overview</span></a></li>
          <li><a class="dropdown-item dropdown-item-active" href="#"> <span>Model</span> </a></li>
          <li><a class="dropdown-item" href="../ML_Results/"> <span>Results</span> </a></li>
          <li><a class="dropdown-item" href="../ML_Impact/"> <span>Impact</span> </a></li>
          </ul>
       </li>
            <li class="nav-item">
              <a class="tab nav-link" href="#">Judges</a>
            </li>
            <li class="nav-item">
              <a class="tab nav-link" href="#">Notebook</a>
            </li>
          </ul>
        </div>
      </nav>
      <div class="header-progress">
        <div class="header-progress-indicator"></div>
      </div>
      
    </header>

    <div class="banner banner-ml">
      <p class="text-center font-size-24">
        <h1 class="banner-h1">Machine Learning</h1>
        <h2 class="banner-h2">Model</h2>
        
        <div class="banner-overlay"></div>
      </p>
    </div>

    <main role="main" class="container wiki">    

      <div class="side-nav">
        <div class="page-title">Model</div>
        <div class="side-nav-menu">
        </div>
      </div>

      <div class="wiki-content">
        
        <h2 class="wiki-h wiki-h2 wiki-section-start" id="blank_1"></h2>
        <p class="wiki-p">
          Machine learning refers to the study of computer algorithms that tend 
          to improve its performance automatically with experience without being 
          explicitly programmed. Over the past decade, machine learning has been 
          applied to perform several complex tasks such as image classification 
          and object recognition. We decided to explore its applications in helping 
          to overcome the problem of antibiotic resistance, a major threat to the 
          human population. The availability of large public datasets makes it 
          crucial and important to utilize the power of machine learning in 
          understanding and predicting biological phenomena such as antibiotic resistance.
        </p>

        <p class="wiki-p">
          To this end, we have used a class of machine learning algorithms called 
          Support Vector Machines in understanding and uncovering the genetic 
          interaction when a bacterial strain is treated with a particular antibiotic. 
          We identified Acinetobacter baumannii as  a critical priority pathogen according 
          to  the World Health Organisation as our target pathogen of interest.
        </p>
        
        <div class="wiki-graphic">
          <img src="../assets/images/images/ML_model_higher.png" alt="ML_Model" class="wiki-graphic-image">
        </div>
        <br><br><br><br><br>

        <h3 class="wiki-h wiki-h3 wiki-section-start" id="step_1">Step 1: Data Collection</h3>
        <p class="wiki-p">
          We utilized and selected 1360 strains of <i>A. baumannii</i> whose AMR phenotypic data was available 
          in the PATRIC database [1]. The testing data includes the outcome when the strain is treated 
          with a particular antibiotic. The outcome can be binary i.e. strain can be either resistant or 
          susceptible to the antibiotic. Strains with verified phenotypes from laboratory data were selected 
          for the analysis, thus excluding strains that were validated only via computational methods. 
        </p>

        <p class="wiki-p">
          We choose 10 different antibiotics to understand the influence of genetic information on 
          resistance phenotype  when different strains are treated with these drugs. The different 
          antibiotics are as follows,
        </p>

        <div class="table-responsive">
          <table class="table table-hover table-bordered">
            <thead class="thead-dark">
              <tr>
                <th scope="col">Antibiotic</th>
                <th scope="col">Mechanism</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>Ciprofloxacin</td>
                <td rowspan="2">DNA Replication</td>
              </tr>

              <tr>
                <td>Levofloxacin</td>
              </tr>



              <tr>
                <td>Gentamicin</td>
                <td rowspan="3">Protein Synthesis</td>
              </tr>

              <tr>
                <td>Tobramycin</td>
              </tr>

              <tr>
                <td>Amikacin</td>
              </tr>




              <tr>
                <td>Ceftriaxone</td>
                <td rowspan="3">Cell Wall Synthesis</td>
              </tr>

              <tr>
                <td>Imipenem</td>
              </tr>

              <tr>
                <td>Ceftazidime</td>
              </tr>



              <tr>
                <td>Trimethoprim+Sulfamethoxazole</td>
                <td>Folate disruption</td>
              </tr>



              <tr>
                <td>Ampicillin+Sulbactam</td>
                <td>Cell Wall Synthesis</td>
              </tr>
            </tbody>
          </table>
        </div><br><br><br><br>


        <h3 class="wiki-h wiki-h3 wiki-section-start" id="step_2">Step 2: Genome Annotation</h3>
        <p class="wiki-p">
          The genomes of the strains were then annotated to develop a pan-genome which is the 
          entire set of genes present in all the selected strains. The genome annotation was 
          carried out using Prokka software [2] which is used for prokaryotic genome annotations. 
          This software was able to identify and annotate alleles as well as their respective genes. 
          The software is publicly available at, 
          <a href="https://github.com/tseemann/prokka" class="wiki-a">https://github.com/tseemann/prokka</a>. 
          We utilized the bioconda channel of the conda environment to run the software. 
        </p><br><br><br><br>


        <h3 class="wiki-h wiki-h3 wiki-section-start" id="step_3">Step 3: Binarization</h3>
        <p class="wiki-p">
          After the formation of the pan-genome and getting the list of all genes and alleles present 
          in all the strains, we created a binary matrix with each row representing a particular strain 
          and each column representing a particular gene/allele. If a particular strain has that particular 
          gene/allele, the value at that position in the matrix is 1, else 0. In simpler terms, if there 
          are ‘n’ number of genes/alleles in the pan-genome, we represent each strain as a vector of ‘n’ 
          dimensions wherein a particular index of the vector refers to a gene/allele. The value at a 
          particular index if gene/allele corresponding to that index is present in the strain. 
          The strains are referred to as examples, while genes/alleles are referred to as features. 
        </p>
        <p class="wiki-p">
          Along with representing the strains in terms of binary vectors, 
          we also collected the phenotype information of strains for a particular 
          antibiotic. So at this stage, we have vector representation of strain i.e. 
          input and phenotype of strain i.e. output ready. The machine learning algorithm 
          will be developed to predict the phenotype of strain using gene/allele vector 
          representation of strains. We have used Support Vector Machines (SVMs) as a 
          machine learning algorithm. 
        </p><br><br><br><br>


        <h3 class="wiki-h wiki-h3 wiki-section-start" id="step_4">Step 4: SVM Training</h3>
        <h5 class="wiki-h wiki-h5">Support Vector Machines (SVM)</h5>
        <p class="wiki-p">
          SVM [3] is a supervised machine learning algorithm which is mainly used for analyzing the 
          data for the classification task. The algorithm represents all the examples of different 
          labels in higher-dimensional space, the number of dimensions of this space is usually the 
          number of features which are the number of genes/alleles in this case. Here in our case, 
          the SVM algorithm represents the strains in ‘n’ dimensional space where ‘n’ is the number 
          of genes/alleles in the pan-genome. Each dimension represents a particular gene/allele 
        </p>
        <p class="wiki-p">
          After representing the strains in ‘n’ dimensional space, the algorithm tends to find 
          the most optimal plane which can differentiate between both labels i.e. Resistant and 
          Susceptible. This optimal plane is also referred to as a hyperplane. The hyperplane is 
          constructed such that the distance between the hyperplane and the nearest example 
          represented in the space is maximized.<br> 
          The illustration about the working of SVM is shown in the above figure, wherein 
          the samples are represented in two-dimensional space using two alleles for the sake of simplicity . 
          However in reality, the space is occupied in ‘n’ dimensions. Different hyperplanes are shown 
          which act as the decision boundary for predicting the phenotype, i.e. labels on either side 
          of this boundary will be different. The decision boundary can never be perfect but SVM tries 
          to achieve the most optimal decision boundary based on the examples given.
        </p><br><br><br><br>


        <h3 class="wiki-h wiki-h3 wiki-section-start" id="step5">Step 5: Computation of Weights</h3>
        <p class="wiki-p">
          The type of SVM algorithm used in our case is Linear SVM i.e. the hyperplane is a 
          linear boundary or the hyperplane is a linear function of features. As in our case, 
          there are ‘n’ features representing a particular strain (or example), so the equation 
          of the hyperplane will be linear and can be represented as,
        </p>

        <div class="wiki-graphic">
          <img src="../assets/images/images/ML_Model_weight_equation_higher.png" alt="Equation" class="wiki-graphic-image">
          <div class="wiki-graphic-reference">wherein, xi refers to the ith gene/allele and wi refers to the linear coefficient of the ith gene/allele</div>
        </div>

        <p class="wiki-p">
          This linear coefficient is referred to as the weight of the particular gene/allele and it represents the 
          quantitative  weightage given to the presence/absence of a particular gene/allele while making predictions. 
          The linear coefficient can be positive or negative, wherein the +/- sign decides the impact of the gene/allele 
          on the final prediction. 
        </p>

        <p class="wiki-p">
          We trained SVM not for multiple  iterations since machine learning algorithms are 
          probabilistic in nature and they tend to produce a different output each time. 
          Running the algorithm for more and more number of iterations helps in achieving more stable and reliable 
          results. We find the hyperplane for each iteration and from where we calculate the weight of the particular 
          gene/allele and represent them as a matrix as shown in the figure. Each row of the matrix represents a 
          particular gene/allele while each column represents a particular iteration of the process. The value at a 
          particular position refers to the weight of the gene/allele in that row during the iteration number of that column.
        </p><br><br><br><br>


        <h3 class="wiki-h wiki-h3 wiki-section-start" id="step6">Step 6: Top AMR alleles</h3>
        <p class="wiki-p">
          As mentioned above, every gene/allele is given a weightage while developing a hyperplane. The more the 
          magnitude of the weight, the more is the importance of that gene/allele in predicting the phenotype 
          of the strain. Since the sign (+/-) of the value of weight merely indicates the direction of impact 
          of that particular gene/allele i.e. if the sign is negative, it means that the gene/allele is 
          responsible for shifting the prediction to Susceptible and if the sign is positive, it means that 
          the gene/allele is responsible for shifting the predicted phenotype to Resistant. So, it is the 
          magnitude of the weight, which determines the relative importance of different genes/alleles. We 
          calculated the sum of absolute values of weights given to each gene/allele for every iteration. 
          The higher the value of this sum, the higher is the relative importance of that gene/allele. We 
          sorted different genes/alleles in the order of their relative importance based on the sum of 
          absolute weights and found out the list of top AMR genes/alleles. It is not sure that these 
          genes/alleles will confer resistance to the antibiotics, they can confer susceptibility to the 
          antibiotic as well since we have taken the sum of absolute weights neglecting the direction of 
          impact of that gene/allele. It must be noted that the absolute weights have no mathematical, or 
          physical, or biological significance, but only provide us an idea about the relative importance 
          of different weights in predicting resistance or susceptibility. They have no absolute 
          significance but surely relative importance. 
        </p><br><br><br><br>


        <h3 class="wiki-h wiki-h3 wiki-section-start" id="step7">Step 7: Correlation analysis</h3>
        <p class="wiki-p">
          We selected the list of top 40 genes/alleles based on the sum of the absolute weights resulting from the 
          iterations. Now, since we have the weights of these genes/alleles for each iteration as well, we calculate 
          the pairwise correlation between the weights of these top 40 genes/alleles. For example let us suppose, 
          there are ‘k’ iterations, then every gene/allele will have a ‘k’ number of weights i.e. it can be represented 
          as the vector of ‘k’ dimensions. For finding a correlation between two genes/alleles, we calculated the Pearson 
          correlation between their corresponding vectors. The positive correlation would mean that an increase in weights 
          of a particular gene/allele is accompanied by the increase in weights of another gene/allele and vice versa.
        </p>
        <p class="wiki-p">
          These correlation analyses provide us with an idea of the relationship between two genes/alleles which 
          is further explored while analyzing the impact of a mutation in particular genes on the resistance 
          phenotype of the strain.
        </p><br><br><br><br>


        <h3 class="wiki-h wiki-h3 wiki-section-start" id="step8">Step 8: Mutational Analyses</h3>
        <p class="wiki-p">
          Like in the case of weights given to a particular gene/allele, the sign of the value was merely an indicator 
          of the direction of its impact on resistance phenotype, similarly, the sign in the case of correlation 
          between two genes/alleles is also an indicator of the direction of variation of their weights. We 
          selected the top pair of genes/alleles based on the magnitude of their correlation and analyzed them 
          for the impact of a mutation in the respective genes on the resistant phenotype. We mainly look for 
          the cases, for example where a mutation in gene A was responsible for resistance to a particular 
          antibiotic, but not in the case when another gene B was also present along with mutated gene A. 
          We performed these analyses for the pairs with the highest correlation values. These help us to 
          make better conclusions about the relationship between a particular pair of genes/alleles.
        </p><br><br><br><br>

        <h3 class="wiki-h wiki-h3 wiki-section-start" id="References">References</h3>
        <p class="wiki-p">
          <ol class="wiki-ol">

            <li>PATRIC, the bacterial bioinformatics database and analysis resource, 
              Wattam A. R. et al., Nucleic Acids Research, Database issue (42), 
              D581-D591 (2013),  
              <a href="https://doi.org/10.1093/nar/gkt1099" class="wiki-a">DOI: 10.1093/nar/gkt1099</a>
            </li>

            <li>Prokka: rapid prokaryotic genome annotation, Seemann T., Bioinformatics, 30(14):2068-9 (2014), 
              <a href="https://doi.org/10.1093/bioinformatics/btu153" class="wiki-a">DOI: 10.1093/bioinformatics/btu153</a>
            </li>

            <li>Support-Vector Networks, Cortes C., Vapnik V., Machine Learning, 20, 273-297 (1995),  
              <a href="https://doi.org/10.1007/BF00994018" class="wiki-a">DOI: 10.1007/BF00994018</a>
            </li>

          </ol>
        </p><br><br><br><br><br>



      </div>
    </main>

    <div class="fab-border"></div>
    <a class="fab" href="#">
      <img src="../assets/images/icons/next.svg" data-toggle="tooltip" data-placement="left" title="Next section" class="fab-next-section"></img>
      <img src="../assets/images/icons/top.svg" data-toggle="tooltip" data-placement="left" title="Top"  class="fab-top-section"></img>
    </a>
    <div class="circular-progress-container">
      <svg class="progress-svg" viewbox="0 0 84 84">
        <circle class="progress-pie" cx="42" cy="42" r="40"></circle>
      </svg>
    </div>

    <footer class="footer">
      <div class="footer-brand"><a href="#" class="">iGEM IIT Roorkee</a></div>
      <div class="footer-lists">
        <div class="footer-list">
          <div class="footer-list-title">People</div>
          <a href="../Team/" class="tab footer-list-item"><span>Team</span></a>
          <a href="../Attributions/" class="tab footer-list-item"><span>Attributions</span></a>
          <a href="../Sponsors/" class="tab footer-list-item"><span>Sponsors</span></a>
        </div>
        <div class="footer-list">
          <div class="footer-list-title">Project</div>
          <a href="../Description/" class="tab footer-list-item"><span>Description</span></a>
          <a href="../Problem/" class="tab footer-list-item"><span>Problem</span></a>
          <a href="../Design/" class="tab footer-list-item"><span>Design</span></a>
          <a href="../Engineering/" class="tab footer-list-item"><span>Engineering</span></a>
          <a href="../Implementation/" class="tab footer-list-item"><span>Implementation</span></a>
          <a href="../Contribution/" class="tab footer-list-item"><span>Contributions</span></a>
        </div>
        <div class="footer-list">
          <div class="footer-list-title">Human Practices</div>
          <a href="../Human_Practices/" class="tab footer-list-item"><span>Overview</span></a>
          <a href="../Education/" class="tab footer-list-item"><span>Science Communication and Public Engagement</span></a>
          <a href="../Integrated_HP/" class="tab footer-list-item"><span>Integrated Human Practices</span></a>
          <a href="../Sustainable/" class="tab footer-list-item"><span>Sustainable Development Goals</span></a>
          <a href="../Collaborations/" class="tab footer-list-item"><span>Collaborations</span></a>
        </div>
        <div class="footer-list">
          <div class="footer-list-title">Dry Lab</div>
          <a href="../Model/" class="tab footer-list-item"><span>Modelling</span></a>
          <a href="../DryLab_Results/" class="tab footer-list-item"><span>Results</span></a>
          <a href="../Software/" class="tab footer-list-item"><span>Software</span></a>
        </div>
        <div class="footer-list">
          <div class="footer-list-title">Wet Lab</div>
          <a href="../Experiments/" class="tab footer-list-item"><span>Overview</span></a>
          <a href="../Results/" class="tab footer-list-item"><span>Results</span></a>
          <a href="../Protocols/" class="tab footer-list-item"><span>Protocols</span></a>
          <a href="../Parts/" class="tab footer-list-item"><span>Parts</span></a>
          <a href="../Safety/" class="tab footer-list-item"><span>Safety</span></a>
          <a href="../References/" class="tab footer-list-item"><span>References</span></a>
        </div>
        <div class="footer-list">
          <div class="footer-list-title">Machine Learning</div>
          <a href="../ML_Overview/" class="tab footer-list-item"><span>Overview</span></a>
          <a href="#" class="tab footer-list-item footer-list-item-active"><span>Model</span></a>
          <a href="../ML_Results/" class="tab footer-list-item"><span>Results</span></a>
          <a href="../ML_Impact/" class="tab footer-list-item"><span>Impact</span></a>
        </div>
      </div>
      <div class="footer-info">
        <div class="footer-info-address">
          <div class="footer-address">
            The Department of Biotechnology, <br />
            Indian Institute of Technology, Roorkee <br />
            Roorkee, Uttarakhand, <br />
            India - 247667
            </div>
          <img src="../assets/images/images/james_thomason_building.svg" class="footer-james-thomason"></img>
        </div>
        <div class="footer-info-socials">
          <div class="footer-section-heading">Contact us:</div>
          <div class="contacts">
            <a href="tel:+011-2875468" class="contact"><span>+011-2875468</span></a>
            <a href="tel:+91-76487699" class="contact"><span>+91-76487699</span></a>
            <a href="mailto:igem@iitr.ac.in" class="contact"><span>igem@iitr.ac.in</span></a>
          </div>
          <div class="footer-section-heading">Follow us on:</div>
          <div class="socials">
            <a href="https://www.instagram.com/igem_iitr/" class="social"><img src="../assets/images/icons/instagram.svg" alt="Instagrm"></a>
            <a href="https://www.facebook.com/IGEM-IIT-Roorkee-111403760578466" class="social"><img src="../assets/images/icons/facebook.svg" alt="Facebook"></a>
            <a href="https://twitter.com/igem_iitr" class="social"><img src="../assets/images/icons/twitter.svg" alt="Twitter"></a>
          </div>
        </div>
        <img src="../assets/images/images/james_thomason_building.svg" class="footer-james-thomason-mobile"></img>
      </div>
    </footer>

    <script src="../js/igem-reset.js"></script>
    <script src="../js/jquery-slim.min.js"></script>
    </script>
    <script src="../js/jquery.waypoints.min.js"></script>
    <script src="../js/popper.min.js"></script>
    <script src="../js/bootstrap.min.js"></script>
    <script src="../js/holder.min.js"></script>
    <script src="../js/index.js"></script>
    <script>
      Holder.addTheme("thumb", {
        bg: "#55595c",
        fg: "#eceeef",
        text: "Thumbnail",
      });
    </script>
  </body>
</html>
