<html>
  <head>
    <!-- <link rel="stylesheet" href="../css/styles.css"> -->
    <style>
      .poster-content-image {
        margin: 10px;
      }
      .poster-content-image-full {
        display: block;
        margin-left: auto;
        margin-right: auto;
        max-width: 70%;
      }
      .sponsor-images {
        flex-flow: wrap;
        text-align: center;
      }
      .sponsor-images img {
        margin: 10px;
        width: 155px;
      }
      .collab-images {
        flex-flow: wrap;
        text-align: center;
      }
      .collab-images img {
        margin: 10px;
        width: 155px;
      }
      .flex-row {
        display: flex;
        flex-direction: row;
      }
      .flex-column {
        display: flex;
        flex-direction: column;
      }
      .img-third {
        width: 30%;
        min-width: 160px;
        max-width: 160px;
        margin: auto 10px;
      }
      .row-third {
        width: 28%;
        margin: 10px;
        text-align: left;
      }
      table,
      th,
      td {
        border: 1px solid black;
        border-collapse: collapse;
      }
      td {
        text-align: center;
      }
      * {
        font-family: "Montserrat" !important;
      }

      ::-moz-selection {
        /* Code for Firefox */
        background: #7bb8fa50 !important;
      }

      ::selection {
        background: #7bb8fa50 !important;
      }

      /* FONTS */
      @font-face {
        font-family: "Montserrat";
        font-style: normal;
        font-weight: 300;
        src: local("Montserrat Light"), local("Montserrat-Light"),
          url(../assets/fonts/Montserrat/Montserrat-Light.ttf)
            format("truetype");
        font-display: swap;
      }
      @font-face {
        font-family: "Montserrat";
        font-style: normal;
        font-weight: 400;
        src: local("Montserrat Regular"), local("Montserrat-Regular"),
          url(../assets/fonts/Montserrat/Montserrat-Regular.ttf)
            format("truetype");
        font-display: swap;
      }
      @font-face {
        font-family: "Montserrat";
        font-style: normal;
        font-weight: 500;
        src: local("Montserrat Medium"), local("Montserrat-Medium"),
          url(../assets/fonts/Montserrat/Montserrat-Medium.ttf)
            format("truetype");
        font-display: swap;
      }
      @font-face {
        font-family: "Montserrat";
        font-style: normal;
        font-weight: 600;
        src: local("Montserrat SemiBold"), local("Montserrat-SemiBold"),
          url(../assets/fonts/Montserrat/Montserrat-SemiBold.ttf)
            format("truetype");
        font-display: swap;
      }
      @font-face {
        font-family: "Montserrat";
        font-style: normal;
        font-weight: 700;
        src: local("Montserrat Bold"), local("Montserrat-Bold"),
          url(../assets/fonts/Montserrat/Montserrat-Bold.ttf) format("truetype");
        font-display: swap;
      }
      @font-face {
        font-family: "Montserrat";
        font-style: normal;
        font-weight: 800;
        src: local("Montserrat Bold"), local("Montserrat-Bold"),
          url(../assets/fonts/Montserrat/Montserrat-ExtraBold.ttf)
            format("truetype");
        font-display: swap;
      }
      /* FONTS END */
    </style>
  </head>
  <body>
    <div class="page_wrapper">
      <div class="poster_w content">
        <div class="row">
          <div class="section full first_text" id="title">
            <img src="../assets/images/poster/Head.png" />
            <div class="info">
              <div class="title">
                Pyomancer: Novel Antibacterial Protein Complexes for MDR
                Infections
              </div>
              <div class="text">
                <h5>Presented by: Team IIT Roorkee</h5>
                <br />
                Team: Sanjeevani Marcha<sup>1</sup>, Muskaan
                Bhambri<sup>1</sup>, Harkirat Singh Arora<sup>1</sup>, Kushagra
                Rustagi<sup>1</sup>, Pradum Kumar<sup>1</sup>, Yash
                Aggrawal<sup>1</sup>, Nitish Verma<sup>1</sup>, Siddharth
                FItwe<sup>1</sup>, Tishee Natani<sup>1</sup>, Kartikey
                Kansal<sup>1</sup>, Kanishk Sugotra<sup>1</sup>, Lakshya
                Jain<sup>1</sup>, Mihir Sachdeva<sup>1</sup>, Dr Siva Ram
                Uppalapati<sup>2</sup>, Somok Bhowmik<sup>2</sup>, Professor
                Naveen K. Navani<sup>2</sup>, Professor Ranjana Pathania<sup
                  >3</sup
                >
                <br />
                <h4><sup>1</sup>iGEM Student team member</h4>
                <h4><sup>2</sup>iGEM Advisor</h4>
                <h4><sup>3</sup>iGEM Primary PI</h4>
                <br /><br />
                <b>Abstract</b>
                <br />
                Our project is aimed at creating novel antibacterial protein
                complexes called Seekercins, designed to specifically bind and
                kill species of multidrug-resistant bacteria. This would be
                achieved by the fusion of antibacterial proteins called pyocins
                and bacteriophage tail fibers. Furthermore, it aims to create a
                system for the easy production of such antimicrobials for the
                use of the iGEM community, consisting of a collection of parts
                and software tool. ML algorithms were used to uncover
                relationships between resistance genes so as to resensitize them
                to antibiotics. Validation will be done experimentally,
                targeting the pathogen <i>Acinetobacter baumannii</i>, and
                production will be done in <i>E. coli.</i> <br /><br />
              </div>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="section full" id="introduction">
            <img src="../assets/images/poster/Intro.png" />

            <div class="info">
              <div class="title">Introduction</div>
              <div class="text">
                Hospital-acquired infections are a leading cause of death
                worldwide with 1.4 million active infections every year. In
                India, the infection rate is alarming and is estimated at about
                35-40% of all hospital admissions. With around 49 million cases
                of COVID-19 so far, the coronavirus pandemic is superimposed on
                the growing pandemic of drug resistant bacteria.
                Ventilator-associated pneumonia (VAPs) are one of the most
                severe comorbidities with ICU mortality rate of
                <b>45.6% to 60.9%</b>, reaching <b>84.3%</b> when caused by
                <b>XDR <i>A. baumannii</i></b
                >. <br /><br />
                As a solution, we use pyocins, which are high molecular
                weight bactericidal protein complexes produced by
                <i>P. aeruginosa</i>. Pyocins attach to receptors on the
                bacterial cell wall with tail fibers and penetrate it with the
                hollow tube, leading to dissipation of membrane potential and
                subsequent cell death. They have a high killing efficiency but a
                very narrow host spectrum, and are similar in structure to
                Myovirus bacteriophages. <br /><br />
                <img
                  src="../assets/images/poster-content/poster_intro1.png"
                  class="poster-content-image poster-content-image-full"
                />
              </div>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="section half" id="inspiration-and-solution">
            <img src="../assets/images/poster/Inspiration and Sol..png" />
            <div class="info">
              <div class="title">Inspiration & Solution</div>
              <div class="text">
                <h3>Inspiration</h3>
                Interviews with doctors and ICU specialists made us realise
                the necessity for alternative treatments for drug
                resistant infections in the face of rapid AMR development.
                <br /><br />
                Our key inspiration was TAU Israel’s 2019 iGEM Project, Pyo-Pyo.
                They illustrated the similarity of pyocins and phages which led
                us to improve upon their strategy and software in our project.
                Heightened sensitivity to hygiene during the pandemic allowed us
                to disseminate WHO’s Global Action Plan against AMR. Nationwide
                lockdown due to the COVID-19 pandemic inspired us to explore
                protein modelling, development of software and machine learning
                tools.

                <br /><br />

                <h3>Solution</h3>

                Our solution is the development of ‘Seekercins’, novel
                antibacterial protein complexes by engineering of R-type pyocins
                to target specific drug resistant bacteria. R-type pyocins have
                a restricted antibacterial spectrum and here we exploit their
                similarity to phages to effectively retarget them. We made
                fusion tail fibres which mediate the binding to the receptor
                from genes of pyocin and phage tail fibers. The gene cluster
                will be cloned into an
                <i>E. coli</i> expression vector under an inducible promoter. We
                plan to purify the Seekercins and perform antimicrobial assays
                against MDR <i>A. baumannii</i>
                strains and biochemical characterization.
                <br /><br />
                <img
                  src="../assets/images/poster-content/poster_inspiration_solution_1.png"
                  class="poster-content-image poster-content-image-full"
                />
              </div>
            </div>
          </div>

          <div class="section half" id="engineering">
            <img src="../assets/images/poster/Engineering.png" />

            <div class="info">
              <div class="title">Engineering</div>
              <div class="text">
                Principles of seekercin design-
                <ol class="wiki-ol">
                  <li class="wiki-li">
                    All native proteins of the pyocin tail structure including
                    tube, spike, sheath and baseplate must be conserved for
                    proper functioning
                  </li>
                  <li class="wiki-li">
                    The initial 164 N-terminal amino acid residues of the
                    wild-type pyocin tail fiber must be conserved for proper
                    functioning
                  </li>
                  <li class="wiki-li">
                    C-terminal of the fusion tail fiber (originating from the
                    phage-tail) must bind specifically to a surface-accessible
                    receptor(sugar or protein) on the cell wall of the target
                    bacterium
                  </li>
                </ol>

                <br /><br />

                After literature review of existing
                <i>Acinetobacter baumannii</i>
                phages, phage AP22 was selected as a suitable candidate for its
                large lytic spectrum.

                <br /><br />

                Rational design of fusion tail fiber-

                <br /><br />

                Analysis of key domains in AP22 tail fiber (Fig 4) revealed a
                conserved lectin-fold similar to that of R2-pyocin tail fiber.
                C-terminal sequence alignment revealed that the last 137 amino
                acids of AP22 constituting the head and shaft of AP22 tail fiber
                aligned with the last 134 amino acids of R2 pyocin tail fiber.
                We replaced the lectin-fold head region of R-pyocin tail fiber
                with a similar lectin-fold binding domain from
                <i>A. baumannii</i>
                specific bacteriophage AP22 tail fiber which naturally targets
                these bacteria. Models of the fusion tail fiber were verified
                for integrity.

                <br /><br />
                <img
                  src="../assets/images/poster-content/Poster_engineering1.png"
                  class="poster-content-image poster-content-image-full"
                />

                <br /><br />
                The gene responsible for the tail fiber from the wild-type
                pyocin gene cluster, PA0620 has been replaced by the fusion tail
                fiber. The gene cluster and fusion tail fiber was synthesized by
                and received from IDT.

                <br /><br />

                <img
                  src="../assets/images/poster-content/poster_engineering2.png"
                  class="poster-content-image poster-content-image-full"
                />

                <br /><br />

                As a chassis we chose to produce them in
                <i>Escherichia coli</i>, strain BL21(DE3).As an
                expression vector, we chose pET-28 a from Novagen which contains
                a T7 promoter and <i>lac operator</i>, allowing
                IPTG-induced protein expression.

                <br /><br />

                <img
                  src="../assets/images/poster-content/poster_engineering3.png"
                  class="poster-content-image poster-content-image-full"
                />

                <br /><br />
              </div>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="section two_thirds" id="model">
            <img src="../assets/images/poster/Model.png" />
            <div class="info">
              <div class="title">Model</div>
              <div class="text">
                We modeled our protein using all the three bioinformatics-based
                modeling techniques. 11 models obtained through these methods
                were compared and the best one was chosen.

                <br /><br />

                <div class="flex-row">
                  <div class="img-third">
                    <img
                      src="../assets/images/poster-content/poster_model_1.jpg"
                      alt=""
                    />
                  </div>
                  <div class="">
                    <ul class="wiki-ul">
                      <li class="wiki-li">
                        Used when we have a structurally unknown protein and a
                        structurally similar known protein
                      </li>
                      <li class="wiki-li">
                        Performed using Swiss-Model web server
                      </li>
                      <li class="wiki-li">Got 1 predicted model</li>
                    </ul>
                  </div>
                </div>
                <br /><br />
                <div class="flex-row">
                  <div class="img-third">
                    <img
                      src="../assets/images/poster-content/poster_model2.jpg"
                      alt=""
                    />
                  </div>
                  <div class="">
                    <ul class="wiki-ul">
                      <li class="wiki-li">
                        Uses protein folds of similar proteins found in
                        different databases to predict the structure of the
                        unknown protein
                      </li>
                      <li class="wiki-li">
                        Performed using I-Tasser web server
                      </li>
                      <li class="wiki-li">Got 5 predicted models</li>
                    </ul>
                  </div>
                </div>
                <br /><br />
                <div class="flex-row">
                  <div class="img-third">
                    <img
                      src="../assets/images/poster-content/poster_model3.jpg"
                      alt=""
                    />
                  </div>
                  <div class="">
                    <ul class="wiki-ul">
                      <li class="wiki-li">
                        Used when structural information is not available for a
                        similar protein
                      </li>
                      <li class="wiki-li">
                        Performed using Robetta Baker Lab web server
                      </li>
                      <li class="wiki-li">Got 5 predicted models</li>
                    </ul>
                  </div>
                </div>
              </div>
            </div>
          </div>

          <div class="section third" id="parts">
            <img src="../assets/images/poster/Parts.png" />
            <div class="info">
              <div class="title">Parts</div>
              <div class="text">
                <b>BASIC PARTS:</b>

                <br />

                We have introduced two new basic Parts to the iGEM registry as a
                contribution for us future iGEM teams.

                <br />

                <img
                  src="../assets/images/poster-content/poster_Parts_1.png"
                  alt=""
                />

                <br />

                <p>
                  <b>Part: BBa_K3564001 :</b>Gene cluster encoding for R-type
                  pyocin with the tail fiber gene deleted
                </p>
                <p>
                  <b>Part: BBa_K3564002 :</b>Fusion tail fiber of R pyocin tail
                  fiber and Acinetobacter phage AP22 tail fiber
                </p>

                <br />

                <b>COMPOSITE PARTS:</b>

                <br />

                <img
                  src="../assets/images/poster-content/Poster_Parts_2.png"
                  alt=""
                />

                <br />

                Part:BBa_K3564102 : Composite of BBa_K3564001 and BBa_K3564002
                that expresses a Seekercin, i.e. a pyocin targeted to
                specifically Bind and kill <i>Acinetobacter baumannii</i>
                <br /><br>
                Characterization for the composite parts would be updated once
                the experiments planned are completed post-pandemic.
              </div>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="section half" id="ihp">
            <img src="../assets/images/poster/IHP.png" />
            <div class="info">
              <div class="title">Integrated Human Practices</div>
              <div class="text">
                <div class="flex-row">
                  <div class="img-third">
                    <img
                      src="../assets/images/poster-content/Poster_IHP 1.png"
                      alt=""
                    />
                  </div>
                  <div class="">
                    <p><b> Dr. Nand Kishore Joshi </b></p>
                    <ul class="wiki-ul">
                      <li class="wiki-li">
                        The team received insightful facts on the situation of
                        AMR infections inside ICUs during COVID-19.
                      </li>
                      <li class="wiki-li">
                        We got to know that doctors use IL-6 antagonists and
                        immunosuppressants for COVID Regulation.
                      </li>
                      <li class="wiki-li">
                        But they observed that the patient became more
                        susceptible to deadly bacterial infections, which are
                        mostly XDR/MDR inside the ICU, which increased the need
                        for a solution.
                      </li>
                    </ul>
                  </div>
                </div>
                <br /><br /><br />
                <div class="flex-row">
                  <div class="img-third">
                    <img
                      src="../assets/images/poster-content/Poster_IHP 2.png"
                      alt=""
                    />
                  </div>
                  <div class="">
                    <p><b> Ms. Khushboo Gupta </b></p>
                    <ul class="wiki-ul">
                      <li class="wiki-li">
                        Our team was intrigued and inspired by the patient's
                        story throughout her battle with MDR Tuberculosis.
                      </li>
                      <li class="wiki-li">
                        We got to know how mental trauma takes a toll on
                        physical health.
                      </li>
                      <li class="wiki-li">
                        We learned that meditation, yoga, a balanced diet, and
                        positive vibes help in speedy recovery.
                      </li>
                    </ul>
                  </div>
                </div>
                <br /><br /><br />
                <div class="flex-row">
                  <div class="img-third">
                    <img
                      src="../assets/images/poster-content/Poster_IHP 3.png"
                      alt=""
                    />
                  </div>
                  <div class="">
                    <p><b> Dr. Prajwal & Dr. Sebastian Dümcke </b></p>
                    <ul class="wiki-ul">
                      <li class="wiki-li">
                        The team had interactive discussion sessions with
                        Clemedi (a Life Sciences Start-up) associates.
                      </li>
                      <li class="wiki-li">
                        We learned about protein modeling and got inputs in
                        designing the protein as well.
                      </li>
                      <li class="wiki-li">
                        Our software team got useful insights on applications of
                        Machine Learning in the project.
                      </li>
                    </ul>
                  </div>
                </div>
              </div>
            </div>
          </div>

          <div class="section half" id="collabs">
            <img src="../assets/images/poster/Collab.png" />

            <div class="info">
              <div class="title">Collaborations</div>
              <div class="text">
                Team iGEM IIT Roorkee understands the importance of working in
                the team and for the team. We truly realize the impact of
                collaborative work projects on any social, technical, or
                cultural issue. With the same inspiration, we collaborated with
                competing iGEM teams to help spread their as well as our cause
                to aware, acknowledge, and educate the population worldwide. We
                reached out to huge masses and successfully increased the
                awareness quotient of people to an impressive 35 - 40% extent.
                <div class="collab-images">
                  <img
                    src="../assets/images/poster-content/Poster_Collab1.PNG"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/Poster_Collab2.png"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/Poster_Collab3.png"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/Poster_Collab4.png"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/Poster_Collab5.png"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/Poster_Collab6.png"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/Poster_Collab7.png"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/Poster_Collab8.png"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/Poster_Collab9.png"
                    alt=""
                  />
                </div>
              </div>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="section third" id="sdg">
            <img src="../assets/images/poster/SDG.png" />
            <div class="info">
              <div class="title">Sustainable Development Goals</div>
              <div class="text" style="text-align: center">
                <b> OBSERVE. UNDERSTAND. ACT. SUSTAIN. </b>
                <br /><br />
                Establishing peace and prosperity requires efforts from not one
                but every human being on the planet. With this achieved, we can
                eradicate all other issues prevalent globally, like poverty,
                poor health, hunger, inequality, poor usage of resources,
                deteriorating climatic conditions, injustice, and so on. To
                create an impact and eradicate the above-said problems, the
                United Nations (UN) came up with 17 focused SDGs to be achieved
                by 2030. So playing our part, we have contributed to three of
                such goals (as mentioned below) through our project Pyomancer.
                <br /><br />
                <div class="flex-row" style="text-align: center">
                  <div class="row-third">
                    <img
                      src="../assets/images/poster-content/SDG1.png"
                      alt=""
                    />
                    <br />
                    <b>Target 3.2</b>
                    <br />
                    <ul class="wiki-ul">
                      <li class="wiki-li">
                        Providing a revolutionized treatment for HAIs
                      </li>
                      <li class="wiki-li">
                        Running Awareness program on Social Media
                      </li>
                    </ul>
                  </div>
                  <div class="row-third">
                    <img
                      src="../assets/images/poster-content/SDG3.png"
                      alt=""
                    />
                    <br />
                    <b>Target 4.5</b>
                    <br />
                    <ul class="wiki-ul">
                      <li class="wiki-li">
                        Educating children, young adults, and the general public
                        about AMR, Synthetic Biology, and Technology
                      </li>
                    </ul>
                  </div>
                  <div class="row-third">
                    <img
                      src="../assets/images/poster-content/SDG2.jpg"
                      alt=""
                    />
                    <br />
                    <b>Target 17.16 & 17.17</b>
                    <br />
                    <ul class="wiki-ul">
                      <li class="wiki-li">
                        Collaborations with fellow iGEM teams and college groups
                      </li>
                      <li class="wiki-li">
                        Initiated and participated in iGEM x SDGs Projects
                      </li>
                    </ul>
                  </div>
                </div>
              </div>
            </div>
          </div>

          <div class="section two_thirds" id="science_comm">
            <img src="../assets/images/poster/Science Comm.png" />
            <div class="info">
              <div class="title">Science Communication & Public Engagement</div>
              <div class="text">
                <div class="sponsor-images">
                  <img
                    src="../assets/images/poster-content/Poster_SC_PE 1.png"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/poster_SC_PE 2.png"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/poster_SC_PE 3.png"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/Poster_SC_PE 4.png"
                    alt=""
                  />
                </div>
                <br /><br />
                <b>Webinars</b>
                <br />
                Spreading awareness among children and young adults through
                interactive webinar sessions and initiating an open dialogue
                about the multidisciplinary fields.
                <br /><br /><br />
                <b>Surveys</b>
                <br />
                Understanding the real on-ground situation of the hospital
                environment through surveying medical personnel and gauging the
                existing knowledge of AMR & basic Sciences of young adults and
                the general public to level-up the know-how of the issue.
                <br /><br /><br />
                <b>Quiz Competition for the Special Children</b>
                <br />
                Curating learning material for the specially-abled children to
                learn about the possible post-antibiotic era due to the
                Antimicrobial Resistance in an engaging way of Quizzing
                questions.
                <br /><br /><br />
                <b>Machine Learning Workshop</b>
                <br />
                Educating budding researchers about the increasing technological
                use in medicine through a riveting workshop on “Combating AMR
                using Machine Learning”.
                <br /><br /><br />
                <b>Social Media</b>
                <br />
                Influencing youth through trending online platforms like
                Instagram, & facebook; and promoting scientific fraternity and
                social inclusivity.
                <br /><br /><br />
                <b>Policy Review Report</b>
                <br />
                Getting acquainted with the policies and government schemes to
                analyse the contribution of governing bodies towards the
                healthcare industry and the impact that requires undivided
                attention.
              </div>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="section half" id="ml">
            <img src="../assets/images/poster/ML.png" />
            <div class="info">
              <div class="title">Machine Learning</div>
              <div class="text">
                <p><b> Detection of Antibiotic Resistant Genes (DARG) </b></p>
                <br /><br />
                <img
                  class="poster-content-image poster-content-image-full"
                  src="../assets/images/poster-content/poster_ML_1.png"
                  alt=""
                />
                <br />
                Our machine learning approach called Detection of Antibiotic
                Resistant Genes abbreviated as DARG involves eight simple steps-
                beginning with data collection of genomes of different strains
                of <i>A. baumannii</i> along with their response to different
                antibiotics, followed by annotating the genomes to develop
                pan-genome. A particular strain is represented by a binary
                vector wherein 1 signifies the presence of particular allele and
                0 signifying absence of particular allele.
                <br />
                The collected data is passed into an Machine learning algorithm
                called Support Vector Machines followed by interpretation of
                SVM. The SVM algorithm is used to compute the weights given to
                different alleles for predicting response of strains to
                particular antibiotics.
                <br />
                We perform correlation analysis among top alleles, followed by
                analysing the impact of mutations on the resistance of different
                strains.
                <br />
                Machine learning algorithm helps in uncovering genes conferring
                resistance to antibiotics. Different important genes which are
                detected are shown in red along with the mechanism they are
                involved in.
                <br />
                <img
                  class="poster-content-image poster-content-image-full"
                  src="../assets/images/poster-content/poster_ML_2.png"
                  alt=""
                />
                <br />
              </div>
            </div>
          </div>

          <div class="section half" id="software">
            <img src="../assets/images/poster/Software.png" />
            <div class="info">
              <div class="title">Software</div>
              <div class="text">
                <p><b> TailScout </b></p>
                We have developed a web application <b>TailScout</b> which
                generates secondary structure of the Engineered pyocin by taking
                the name of the target bacterium from the user as Input. We have
                used the JPred REST API and Clustal Omega REST API on the Django
                REST framework for the processes. <br /><br />
                <b> Classifying broadly it is a three step process: </b>
                <br />
                1. Phage Tail Fiber Detection using the bacteria name and phage
                tail library.
                <br />
                2. MSA of phage tail fiber sequences for identifying the
                conserved regions using Clustal Omega.
                <br />
                3. Secondary structure prediction of fusion protein using JPred.
                <br />

                Backend Workflow:

                <br /><br />
                <img
                  class="poster-content-image poster-content-image-full"
                  src="../assets/images/poster-content/poster_Software.png"
                  alt=""
                />
              </div>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="section two_thirds" id="implementation">
            <img src="../assets/images/poster/implementation.png" />
            <div class="info">
              <div class="title">Implementation</div>
              <div class="text">
                We suggest a surfactant-based formulation containing a mixture
                of fusion proteins to be used for endotracheal and endobronchial
                administration for the treatment of Ventilator-related pneumonia
                (VAP) is associated with increased stay and mortality in the
                intensive care unit ( ICU) with an estimated 30 % rise in
                underlying disease mortality.
                <br /><br />
                Current strategy is the only way to deal with acquired
                infections is to monitor the spread of infections by
                administering antibiotics. Once the disorder is diagnosed,
                subsequent antibiotics are administered to the sites of
                infections, i.e. alveoli and trachea lining, and we have
                findings for the infection microbial profiles.
                <br /><br />
                We plan to use a surfactant to improve the delivery of the
                medication and achieve a more homogeneous distribution of the
                drug by using the rapid installation of the bolus in conjunction
                with suitable alveolar recruitment techniques. It will require
                the administration of a carefully formulated formulation
                according to each infection's microbial diagnosis. By having a
                unique seekercin for various organisms, we aim to keep our
                therapy specific and targeted, thereby preventing any collateral
                damage and reducing the spectrum of production of resistance.
                <br /><br />
                <br /><br />

                <img
                  src="../assets/images/poster-content/poster_implementation_1.jpg"
                  alt=""
                  class="poster-content-image poster-content-image-full"
                />
                <div class="">
                  Surfactant consists of 80-85 % phospholipids, 5-10 % neutral
                  lipids, and 8-10 % protein, with 5-6 % of the four specific
                  proteins of surfactant. Broadly, the formulation has the
                  following structure:
                </div>
                <br /><br />
                <div class="">
                  <div class="table-responsive">
                    <table class="table table-hover table-bordered">
                      <thead class="thead-dark">
                        <tr>
                          <th scope="col">
                            <b>Mucolytics:</b> Carbocisteine, Erdosteine.
                          </th>
                          <th scope="col">
                            <b>Bronchodilators:</b>Beta-2 agonists,
                            Anticholinergics, Theophylline
                          </th>
                        </tr>
                      </thead>
                      <tbody>
                        <tr>
                          <td>Seekercin A</td>
                          <td>di-Palmitoylphosphatidylcholine (DPPC)</td>
                        </tr>
                        <tr>
                          <td>Seekercin B</td>
                          <td>Phosphatidylglycerol</td>
                        </tr>
                        <tr>
                          <td>Seekercin C</td>
                          <td>Phosphatidylinositol</td>
                        </tr>
                      </tbody>
                    </table>
                  </div>
                </div>
                <br /><br />
                <div class="">
                  By instilling a particular microvolume of liquid into the upper
                  airways and moving the plug by programmed air ventilation into
                  the desired region of the lung, the aqueous liquid containing
                  soluble drugs can be delivered into targeted branches of the
                  lung airway and deposited onto the lung epithelium .
                </div>
                <br /><br />
                <div class="">
                  <img
                    src="../assets/images/poster-content/poster_implementation_2.jpg"
                    alt=""
                    class="poster-content-image poster-content-image-full"
                  />
                </div>
                <br /><br />
                <div class="">
                  We researched and decided on delivery of pyocins through
                  nebulizers however, in India there is a disproportionately large
                  problem with VAPs in the Newborn Intensive Care Units.
                  <br />
                  (NICU) (60k+ deaths per year).
                  <br /><br />
                  On seeking an expert opinion from a Doctor (NICU/VAP
                  specialist), he pointed out that neonates, especially preterm,
                  have underdeveloped lungs that are too delicate for effective
                  nebulizer based treatment.
                  <br /><br />
                  A big drawback of nebulizer therapy is its poor efficiency of
                  delivery, much of the dose is lost in the ETT (mist deposition
                  on the walls, medicine does not get misty well, etc.). Drug loss
                  is no longer a concern when administering the bolus dose, since
                  the substance itself is injected into the lungs, taking the drug
                  with it.
                  <br /><br />
                  As observed in in-vivo studies, our rapid and efficient delivery
                  of bolus-injected surfactant solutions containing
                  surfactant-associated proteins such as SP-A to the alveoli shows
                  a strong precedent for delivery of Seekercin is also more likely
                  to ensure the delivery of high doses (10 ^ 12 pyocins per ml)
                  needed for successful treatment. The dose will easily enter the
                  alveoli, and it is predicted that changes will be observed
                  within seconds to minutes.
                  <br /><br />
                </div>
              </div>
            </div>
          </div>

          <div class="section third" id="future">
            <img src="../assets/images/poster/Future Direction.png" />
            <div class="info">
              <div class="title">Future Direction</div>
              <div class="text">
                <ul class="wiki-ul">
                  <li class="wiki-li">
                    We would complete the planned experiments in time for next
                    year’s iGEM competition as Phase 2 of our project
                  </li>
                  <li class="wiki-li">
                    As Seekercins are protein-based therapeutics, an attempt to
                    reduce their immunogenicity by eliminating relevant epitopes
                    could be made
                  </li>
                  <li class="wiki-li">
                    It is clear from the literature that the number of sequenced
                    phages for pathogenic bacteria are still relatively small.
                    This calls for the need for more extensive isolation,
                    characterizing and sequencing of phages. 
                  </li>
                  <li class="wiki-li">
                    Following positive results in cell line based assays,
                    in-vivo studies in murine or Caenorhabditis models could be
                    done
                  </li>
                  <li class="wiki-li">
                    Comparison of efficacy of Seekercin formulations with
                    various additives in cell line and in-vivo lung infection
                    studies
                  </li>
                  <li class="wiki-li">
                    Studies on efficacy and dosage requirements in
                    multi-Seekercin cocktails for broader strain spectrum.
                  </li>
                  <li class="wiki-li">
                    Increasing the numbers of Multiple Drug-Resistant Bacteria
                    in our phage library.
                  </li>
                  <li class="wiki-li">
                    Incorporating protein modelling of the pyocin bacteriophage
                    fusion protein in features offered by TailScout
                  </li>
                  <li class="wiki-li">
                    Result Analysis of Secondary and Tertiary structure of the
                    fusion protein can also be incorporated into TailScout.
                  </li>
                  <li class="wiki-li">
                    Improving the result and job status retrieval process of
                    TailScout.
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="section half" id="results">
            <img src="../assets/images/poster/results.png" />
            <div class="info">
              <div class="title">Results</div>
              <div class="text">
                <h3 class="wiki-h wiki-h3">Wet Lab</h3>
                <br />
                Initial experiments were performed in March but had to be
                aborted due to the pandemic and closing of our institute. We
                have laid out contingency plans for multiple scenarios based on
                the expected results of our wet lab experiments.
                <br /><br /><br>
                <h3 class="wiki-h wiki-h3">Dry Lab</h3>
                <br />
                <div class="flex-row">
                  <div class="img-third">
                    <img
                      class="poster-content-image poster-content-image-full"
                      src="../assets/images/poster-content/poster_results1.jpg"
                      alt=""
                    />
                  </div>
                  <div class="">
                    <b> Root Mean Square Deviation </b>
                    <br />
                    Calculated and compared the RMSD of each fusion-protein
                    model with:
                    <ul class="wiki-ul">
                      <li class="wiki-li">The R2-NTF tail fibre protein</li>
                      <li class="wiki-li">
                        The AP-22 bacteriophage tail fibre protein
                      </li>
                    </ul>
                  </div>
                </div>
                <br />
                <div class="flex-row">
                  <div class="img-third">
                    <img
                      class="poster-content-image poster-content-image-full"
                      src="../assets/images/poster-content/poster_results2.jpg"
                      alt=""
                    />
                  </div>
                  <div class="">
                    <b> Ramachandran Plot </b>
                    <br />
                    <ul class="wiki-ul">
                      <li class="wiki-li">
                        Gives information about the energetically allowed and
                        disallowed regions in the protein
                      </li>
                      <li class="wiki-li">
                        Prepared and compared plots for 6 models
                      </li>
                    </ul>
                    <br /><br />
                  </div>
                </div>
                <br /><br />
                <div class="table-responsive">
                  <table class="table table-hover table-bordered">
                    <thead class="thead-dark">
                      <tr>
                        <th scope="col">Model/Data</th>
                        <th scope="col">RMSD with lower part (4mtm)</th>
                        <th scope="col">RMSD with upper part (6cu2)</th>
                        <th scope="col">Residues in disallowed regions</th>
                        <th scope="col">Residues in favoured regions</th>
                      </tr>
                    </thead>
                    <tbody>
                      <tr>
                        <td>Ab-initio Model 1 </td>
                        <td>0.435</td>
                        <td>2.648</td>
                        <td>0%</td>
                        <td>89.2%</td>
                      </tr>

                      <tr>
                        <td>Ab-initio Model 2 </td>
                        <td>0.419</td>
                        <td>6.264</td>
                        <td>0%</td>
                        <td>87.8%</td>
                      </tr>

                      <tr>
                        <td>Ab-initio Model 3 </td>
                        <td>0.375</td>
                        <td>2.335</td>
                        <td>0%</td>
                        <td>86.1%</td>
                      </tr>

                      <tr>
                        <td>Ab-initio Model 4 </td>
                        <td>0.475</td>
                        <td>4.024</td>
                        <td>0.2%</td>
                        <td>86%</td>
                      </tr>

                      <tr>
                        <td>Ab-initio Model 5 </td>
                        <td>0.427</td>
                        <td>4.631</td>
                        <td>0</td>
                        <td>89.4%</td>
                      </tr>

                      <tr>
                        <td>Swiss Model</td>
                        <td>4.666</td>
                        <td>1.527</td>
                        <td>0.3%</td>
                        <td>88.1%</td>
                      </tr>
                    </tbody>
                  </table>
                </div>
                <br /><br />

                <img
                  class="poster-content-image poster-content-image-full"
                  src="../assets/images/poster-content/poster_results3.png"
                  alt=""
                />
                <br /><br /><br />

                <b> Best Model → Ab-initio Model 1 </b>
                <ul class="wiki-ul">
                  <li class="wiki-li">
                    After choosing the best model, we performed energy
                    minimization for it to ensure the most stable state
                  </li>
                  <li class="wiki-li">
                    We used YASARA Energy Minimization Toolkit for this
                  </li>
                </ul>
                <br /><br />
                <br />

                <img
                  class="poster-content-image poster-content-image-full"
                  src="../assets/images/poster-content/poster_results4.png"
                  alt=""
                />
                <br />
                <img
                  class="poster-content-image poster-content-image-full"
                  src="../assets/images/poster-content/poster_results5.png"
                  alt=""
                />
              </div>
            </div>
          </div>

          <div class="section half" id="references">
            <img src="../assets/images/poster/Ref. and aknowledegements.png" />

            <div class="info">
              <div class="title">References and Acknowledgements</div>
              <div class="text">
                References:
                <br /><br /><br />
                <ol class="wiki-ol">
                  <li class="wiki-li">
                    <a
                      href="https://www.cdc.gov/hai/organisms/acinetobacter.html"
                    >
                      https://www.cdc.gov/hai/organisms/acinetobacter.html
                    </a>
                  </li>
                  <li class="wiki-li">
                    <a
                      href="https://academic.oup.com/cid/article/42/5/692/2052763"
                    >
                      https://academic.oup.com/cid/article/42/5/692/2052763
                    </a>
                  </li>
                  <li class="wiki-li">
                    <a href="https://2019.igem.org/Team:TAU_Israel">
                      https://2019.igem.org/Team:TAU_Israel
                    </a>
                  </li>
                  <li class="wiki-li">
                    <a
                      href="https://www.researchgate.net/publication/5411516_Retargeting_R-Type_Pyocins_To_Generate_Novel_Bactericidal_Protein_Complexes"
                    >
                      https://www.researchgate.net/publication/5411516_Retargeting_R-Type_Pyocins_To_Generate_Novel_Bactericidal_Protein_Complexes
                    </a>
                  </li>
                  <li class="wiki-li">
                    <a
                      href="https://www.business-standard.com/article/health/ventilator-associated-pneumonia-a-major-cause-of-death-within-icu-in-india-119091400987_1.html"
                    >
                      https://www.business-standard.com/article/health/ventilator-associated-pneumonia-a-major-cause-of-death-within-icu-in-india-119091400987_1.html
                    </a>
                  </li>
                  <li class="wiki-li">
                    <a
                      href="https://www.ijmm.org/article.asp?issn=0255-0857;year=2017;volume=35;issue=4;spage=504;epage=510;aulast=Khurana"
                    >
                      https://www.ijmm.org/article.asp?issn=0255-0857;year=2017;volume=35;issue=4;spage=504;epage=510;aulast=Khurana
                    </a>
                  </li>
                  <li class="wiki-li">
                    <a
                      href="https://www.webmd.com/lung/news/20200415/ventilators-helping-or-harming-covid-19-patients#1"
                    >
                      https://www.webmd.com/lung/news/20200415/ventilators-helping-or-harming-covid-19-patients#1
                    </a>
                  </li>
                  <li class="wiki-li">
                    <a
                      href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3059826/"
                    >
                      https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3059826/
                    </a>
                  </li>
                  <li class="wiki-li">
                    <a href="https://erj.ersjournals.com/content/17/5/1034">
                      https://erj.ersjournals.com/content/17/5/1034
                    </a>
                  </li>
                  <li class="wiki-li">
                    Scholl, D., Cooley, M., Williams, S.R., Gebhart, D., Martin,
                    D., Bates, A. and Mandrell, R., 2009. An engineered R-type
                    pyocin is a highly specific and sensitive bactericidal agent
                    for the food-borne pathogen Escherichia coli O157: H7.
                    Antimicrobial agents and chemotherapy, 53(7), pp.3074-3080.
                  </li>
                  <li class="wiki-li">
                    Scholl, D. and Martin, D.W., 2008. Antibacterial efficacy of
                    R-type pyocins towards <i>Pseudomonas aeruginosa</i> in a
                    murine peritonitis model. Antimicrobial agents and
                    chemotherapy, 52(5), pp.1647-1652.
                  </li>
                  <li class="wiki-li">
                    Williams, S.R., Gebhart, D., Martin, D.W. and Scholl, D.,
                    2008. Retargeting R-type pyocins to generate novel
                    bactericidal protein complexes. Applied and environmental
                    microbiology, 74(12), pp.3868-3876.
                  </li>
                  <li class="wiki-li">
                    Scholl, D., Gebhart, D., Williams, S.R., Bates, A. and
                    Mandrell, R., 2012. Genome sequence of E. coli O104: H4
                    leads to rapid development of a targeted antimicrobial agent
                    against this emerging pathogen. PLoS One, 7(3), p.e33637.
                  </li>
                </ol>
                <br />
                <br /><br />
                Acknowledgements:
                <h4>Darshak Bhatt, Mentor</h4>
                <h4>Vivek Junghare</h4>
                <h4>Student Technical Council, IIT Roorkee</h4>
                <h4>Director IIT Roorkee</h4>
                <h4>Dean SRIC IIT Roorkee</h4>
              </div>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="section full" id="sponsors">
            <img src="../assets/images/poster/Sponsors.png" />
            <div class="info">
              <div class="title">Sponsors</div>
              <div class="text">
                <div class="sponsor-images">
                  <img
                    src="../assets/images/poster-content/poster_sponsors_1.jpg"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/poster_sponsors_2.jpg"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/poster_sponsors_3.jpg"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/poster_sponsors_4.png"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/poster_sponsors_5.jpg"
                    alt=""
                  />
                  <img
                    src="../assets/images/poster-content/poster_sponsors_6.jpeg"
                    alt=""
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>

      <div class="poster_w display">
        <h3 id="section_title"></h3>
        <p id="section_text"></p>
      </div>
    </div>
  </body>
</html>
